packages/expo-router/src/link/elements.tsx

```diff
@@ -125,6 +146,26 @@ export interface LinkMenuProps {
    * Optional SF Symbol displayed alongside the menu item.
    */
   icon?: SFSymbol;
+  /**
+   * Custom image loaded using expo-image's `useImage` hook.
```

Comment:
```suggestion
   * Custom image loaded using `useImage()` hook from `expo-image`.
```

Let's improve readability of these sentences.